packages/main/test/specs/ComboBox.mobile.spec.js

```diff
@@ -86,12 +86,16 @@ describe("Eventing", () => {
 
 		const changeCountText = await browser.$("#change-count").getText();
 		assert.strictEqual(changeCountText, "1", "Change was fired once");
+
+		const dialogOkButton = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover").$(".ui5-responsive-popover-footer").$("ui5-button");
```

Comment:
Why you add an interaction then you don't anything afterwards?
Also the test is failing.